samples/apps/github-qna-webapp-react/src/components/QnA.tsx

```diff
@@ -159,7 +161,18 @@ const QnA: FC<IData> = ({ uri, project, branch, keyConfig, onBack }) => {
                         <Slider min={0} max={1} step={0.1} value={relevance} onChange={onSliderChange} />
                         <Label size="small">1</Label>
                     </div>
-                    <Label size="medium">Relevance: {relevance}</Label>
+                    <InfoLabel
+                        info={
+                            <div style={{ maxWidth: 250 }}>
+                                'Relevance' is used in memory search and is a measure of the relevance score from 0.0 to
```

Comment:
Can we call this "RelevanceThreshold" as a better representation of how this value is being used?

Reply:
Done! Also updated the end labels 
<img width="294" alt="Screenshot 2023-03-21 at 4 24 04 PM" src="https://user-images.githubusercontent.com/125500434/226732049-9d2d8ea2-d696-425e-80cc-e7f4f5a70051.png">
<img width="476" alt="Screenshot 2023-03-21 at 4 24 24 PM" src="https://user-images.githubusercontent.com/125500434/226732119-badf0346-1fd8-408b-ba1c-b73e5770d60b.png">